CHARTER.md

```diff
@@ -0,0 +1,117 @@
+
+
+**Technical** **Charter (the “Charter”)**  
+**for**   
+**Besu a Series of LF Projects, LLC**
+
+**Adopted TBD (date)**
+
+This Charter sets forth the responsibilities and procedures for technical contribution to, and oversight of, the Besu open source project, which has been established as Besu a Series of LF Projects, LLC (the “Project”).  LF Projects, LLC (“LF Projects”) is a Delaware series limited liability company. All contributors (including committers, maintainers, and other technical positions) and other participants in the Project (collectively, “Collaborators”) must comply with the terms of this Charter. 
+
+1. **Mission and Scope of the Project**
+
+   1. The mission of the Project is to develop an Ethereum client designed to be enterprise and infrastructure-friendly for both public and private permissioned network use cases, with an extractable EVM implementation.
+
+   2. The scope of the Project includes collaborative development under the Project License (as defined herein) supporting the mission, including documentation, testing, integration and the creation of other artifacts that aid the development, deployment, operation or adoption of the open source project.
+
+2. **Technical Steering Committee**
+
+   1. The Technical Steering Committee (the “TSC”) will be responsible for all technical oversight of the open source Project. 
```

Comment:
If this is intended to be a Besu-specific TSC (as opposed to a LFDT-level body), I think this needs to be clarified

Reply:
Yes, the TSC is Besu specific.  As a default, you can just set the TSC to be the set of all maintainers and, unless I'm missing something, this will basically exactly model your current governance practices.